docs/source/models.rst

```diff
@@ -745,6 +745,12 @@ produced by :py:func:`mlflow.pytorch.save_model()` and :py:func:`mlflow.pytorch.
 the ``python_function`` flavor, allowing you to load them as generic Python functions for inference
 via :py:func:`mlflow.pyfunc.load_model()`.
 
+.. note::
+    When using the PyTorch flavor, if a GPU is available at prediction time, the default GPU will be used to run
+    inference. To disable this behavior, users can set `MLFLOW_DEFAULT_PREDICTION_DEVICE="cpu"` or explicitly
+    pass in a device with the `device` parameter for the `predict` function. Note that this environment variable is
+    only supported for the PyTorch flavor at this time.
```

Comment:
Can we move this supported frameworks for environment variable documentation to `environment_variables.py` so that we can maintain a list of supported frameworks attached directly to the env var?